assets/xml/objects/object_lightswitch.xml

```diff
@@ -1,11 +1,12 @@
 ﻿<Root>
     <File Name="object_lightswitch" Segment="6">
-        <DList Name="object_lightswitch_DL_000260" Offset="0x260" />
-        <DList Name="object_lightswitch_DL_000398" Offset="0x398" />
-        <DList Name="object_lightswitch_DL_000408" Offset="0x408" />
-        <Texture Name="object_lightswitch_Tex_000420" OutName="tex_000420" Format="rgba16" Width="32" Height="32" Offset="0x420" />
-        <Texture Name="object_lightswitch_Tex_000C20" OutName="tex_000C20" Format="rgba16" Width="32" Height="32" Offset="0xC20" />
-        <Texture Name="object_lightswitch_Tex_001420" OutName="tex_001420" Format="rgba16" Width="32" Height="32" Offset="0x1420" />
-        <Texture Name="object_lightswitch_Tex_001C20" OutName="tex_001C20" Format="rgba16" Width="32" Height="32" Offset="0x1C20" />
+        <DList Name="object_lightswitch_DL_000260" Offset="0x260"/>
+        <DList Name="object_lightswitch_DL_000398" Offset="0x398"/>
+        <!-- a square, geometry only -->
+        <DList Name="object_lightswitch_DL_000408" Offset="0x408"/>
```

Comment:
What changed here? github says something changed, but it seems to be the exact same line, right?

Reply:
```suggestion
        <DList Name="object_lightswitch_DL_000260" Offset="0x260" />
        <DList Name="object_lightswitch_DL_000398" Offset="0x398" />
        <!-- a square, geometry only -->
        <DList Name="object_lightswitch_DL_000408" Offset="0x408" />
```
Looks like we lost the white space between the end of the offset and the `/>`
This seems to apply to all of them in this object.